### DISCUSSION TITLE
Angle mode limited rates

### DISCUSSION THREAD

**Discussion by nsb** [iNavFlight/inav#6768] [Q&A] [answered]
I configured INAV on my Zohd Orbit wing. In manual mode everything is fine but In Angle mode the wing can almost not turn. The rates are very limited. How can I change that?

Diff pasted below.

# diff

# version
# INAV/MATEKF411SE 2.6.1 Feb 25 2021 / 10:43:48 (64249b1a)
# GCC-9.2.1 20191025 (release) [ARM/arm-9-branch revision 277599]

# start the command batch
batch start

# resources

# mixer

mmix reset

mmix 0  1.000  0.000  0.000  0.000

# servo mix
smix reset

smix 0 3 0 70 0 -1
smix 1 3 1 70 0 -1
smix 2 4 0 70 0 -1
smix 3 4 1 -70 0 -1

# servo

# safehome

# logic

# gvar

# feature
feature MOTOR_STOP
feature GPS
feature PWM_OUTPUT_ENABLE
feature FW_LAUNCH

# beeper
beeper -ON_USB

# map
map TAER

# serial
serial 0 2 115200 57600 0 115200
serial 1 64 115200 57600 0 115200
serial 30 4096 115200 115200 0 115200
serial 31 32 115200 115200 115200 115200

# led

# color

# mode_color

# aux
aux 0 0 0 1300 2100
aux 1 1 1 1250 1475
aux 2 11 1 1550 1750
aux 3 10 1 1800 2000
aux 4 12 1 950 1125
aux 5 37 2 1925 2075

# adjrange

# rxrange

# temp_sensor

# wp
#wp 0 invalid

# osd_layout
osd_layout 0 24 12 2 V

# master
set gyro_hardware_lpf = 188HZ
set acc_hardware = MPU6000
set acczero_x = 75
set acczero_y = -4
set acczero_z = -127
set accgain_x = 4086
set accgain_y = 4095
set accgain_z = 4023
set mag_hardware = NONE
set baro_hardware = BMP280
set pitot_hardware = NONE
set serialrx_provider = FPORT
set serialrx_halfduplex = ON
set motor_pwm_protocol = STANDARD
set failsafe_procedure = RTH
set failsafe_mission = OFF
set align_board_pitch = -20
set align_board_yaw = 900
set platform_type = AIRPLANE
set model_preview_type = 8
set small_angle = 180
set applied_defaults = 3
set gps_sbas_mode = AUTO
set nav_wp_radius = 3000
set nav_rth_allow_landing = NEVER
set nav_rth_altitude = 7000
set nav_fw_cruise_thr = 1700
set nav_fw_max_thr = 1800
set nav_fw_control_smoothness = 2
set name = Orbit

# profile
profile 1

set fw_p_pitch = 15
set fw_i_pitch = 40
set fw_ff_pitch = 0
set fw_p_roll = 15
set fw_i_roll = 30
set fw_ff_roll = 0
set yaw_lpf_hz = 30
set rc_expo = 30
set rc_yaw_expo = 30
set roll_rate = 40
set pitch_rate = 15
set yaw_rate = 9

# battery_profile
battery_profile 1

set battery_capacity = 2200

# end the command batch
batch end

**Comment by JulioCesarMatias** [comment, marked as answer]
hello, increase roll_rate and pitch_rate in stabilize mode, or otherwise, increase the max_angle_inclination_rll and max_angle_inclination_pit of the PID which is 30 degrees by default

**Reply by nsb** [reply]
Thanks, will try it out.

**Reply by nsb** [reply]
@JulioCesarMatias Quick question. How do I change the roll and pitch rate specifically for stabilze mode?

**Reply by avsaase** [reply]
Do you mean acro mode? You can set the rates on the rates subtab on the pid tuning tab. But don't increase them above what your airplane is capable of reaching. Best set them a bit lower.

**Reply by JulioCesarMatias** [reply]
> @JulioCesarMatias Pergunta rápida. Como altero a taxa de rotação e inclinação especificamente para o modo de estabilização?

![INAV-Configurator-PID-Tuning-Rates-Expo](https://user-images.githubusercontent.com/25625982/113317654-f6294d00-92e5-11eb-909f-ed27d36b7d12.jpg)

for the stabilize mode (Angle) you must change the parameters that I highlighted in the image. Your Roll & Pitch expo is very low (set rc_expo = 30), the roll rate is cool (set roll_rate = 40), the pitch rate is low (set pitch_rate = 15) increase to 25

**Reply by avsaase** [reply]
It's best to measure the max roll and pitch rate in manual at the throttle level you fly at the most, and then set the rates to 80-90% of what you measured.

**Reply by JulioCesarMatias** [reply]
> É melhor medir a rotação máxima e a taxa de inclinação no manual no nível do acelerador em que você voa no máximo e, em seguida, definir as taxas para 80-90% do que você mediu.

yeah,perfect
